services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/app/ManagementServicesProducer.java

```diff
@@ -55,6 +56,9 @@ public class ManagementServicesProducer {
     @Inject
     MongoDbBasedCredentialsConfigProperties credentialsServiceProperties;
 
+    @Inject
+    MongoDbBasedHttpServiceConfigOptions mongoDbBasedHttpServiceConfigOptions;
+
```

Comment:
Now that these options are needed in multiple places, I'd rather have a corresponding producer in `org.eclipse.hono.deviceregistry.mongodb.app.ConfigPropertiesProducer` and adapt `HttpServerFactory` accordingly. You can then add a corresponding parameter to this class' `deviceManagementService()` method.

Reply:
I believe you can do the same for the JDBC registry correspondingly.

Reply:
Ok, I get back to this a bit later on better time.

Reply:
Done, hope I understood it correctly what you were requesting to change.

Reply:
You did :-)